.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - run: yarn lint:report
 
       - name: Annotate Code Linting Results
-        uses: ataylorme/eslint-annotate-action@v2
+        uses: ataylorme/eslint-annotate-action@v3
```

Comment:
We have already [reverted this change](https://github.com/customerio/customerio-reactnative/pull/247) in the past because it was breaking the flow.

Reply:
Good call-out. 

Seeing that the v3 git tag was created in april, I think it's safe to say that they will not delete the tag this time, which was the problem that was caused previously. 

CI ran successfully and parsed the eslint file successfully so I think v3 is safe to use.